tlatools/org.lamport.tlatools/test/util/AstNode.java

```diff
@@ -305,12 +339,27 @@ private Kind(String name) {
 		}
 		
 		/**
-		 * Get list of unused AST node kinds; useful for identifying gaps in testing.
+		 * Get list of unused AST node kinds specific to TLA+, not PlusCal;
+		 * useful for identifying gaps in testing.
+		 * 
+		 * @return Kinds that were never constructed during parsing.
+		 */
+		public static EnumSet<Kind> getUnusedTlaPlusNodeKinds() {
+			EnumSet<Kind> unused = Kind.unused.clone();
```

Comment:
Nit: Here (and in `getUnusedPlusCalNodeKinds` a few lines lower down), use `new EnumSet<>(Kind.unused)` instead of `clone()`. Clone is convenient but it's riddled with surprises; see e.g. ["Copy Constructor versus Cloning" in this interview](https://www.artima.com/articles/josh-bloch-on-design) with Josh Bloch, one of the original bigwig Java architects. See also #836.